cmd/entrypoint/testutil_fake_filtersecrets_test.go

```diff
@@ -0,0 +1,117 @@
+package entrypoint
```

Comment:
I'm confused why this file is named `testutil_fake_filtersecrets_test.go` instead of just putting it in `secrets_test.go`.